gslib/commands/mb.py

```diff
@@ -145,7 +147,13 @@
                          "enforced", objects in this bucket cannot be made
                          publicly accessible. Default is "unspecified".
 
-""")
+  --rpo setting          Specifies the `replication setting <https://cloud.google.com/storage/docs/turbo-replication>`_.
+                         Valid values are {rpo_values}. If unspecified,
```

Comment:
My understanding is it's a bit more complicated, which we should probably convey here.

Suggestion: Specifies the `replication setting <https://cloud.google.com/storage/docs/turbo-replication>`_.
This flag is not valid for single-region buckets, and multi-region buckets only accept a value of DEFAULT. Valid values for dual region buckets are {rpo_values}. If unspecified, DEFAULT is applied for dual-region and multi-region buckets.

Reply:
Done.